requirements/defaults.txt

```diff
@@ -40,6 +40,7 @@ pygraphviz==1.5
 pyinstrument==3.1.3
 qrcode==6.1
 requests==2.21.0
+sendgrid-django==4.2.0
```

Comment:
Why is this being added here? This should pretty much only be in `requirements/release.txt`

Reply:
Ohh changed back but it looks like both this and the sslserver move were because of how DEBUG is set: based on an environment variable in project/settings/defaults.py, then separately in /local.py. So if you don't have the environment variable set already, you end up with errors when trying to start the server because having DEBUG set to False during the rest of project/settings/defaults.py makes some incorrect choices based on the local value.

It looks like the invoke scripts also just initialize local.py with DEBUG=True rather than copying local-dist if local doesn't exist - should they do the latter?

We actually use `os.environ.get` a lot throughout defaults.py. So to let other people set up for development, we need to do one of
* add "adding the following environment variables" to docs and/or invoke scripts
* go through and check that in each case any values in local-dist.py correctly override the values in defaults (e.g., would need to override both `EMAIL_FROM_ADDRESS` and `DEFAULT_FROM_EMAIL`)
* use dotenv and provide a local-dist .env template
* restructure settings/__init__.py so it loads necessary local settings *first* (in addition to overriding afterwards?)

Reply:
Per slack discussion, recommend us switching over to using `dotenv`

Reply:
Done! Can you take a look when you have a chance and see if this looks reasonable?

Reply:
see new comments